spec/spec.md

```diff
@@ -0,0 +1,180 @@
+<!--
+SPDX-Copyright: 2019 Free Software Foundation Europe e.V.
+
+SPDX-License-Identifier: CC-BY-SA-4.0
+-->
+
+# REUSE Specification
+
+This specification defines a standardized method for declaring copyright and
+licensing for software projects. The goal of the specification is to have
+unambiguous, machine-readable copyright and licensing information for each
+individual file in a project.
+
+## Definitions
+
+These are the definitions for some of the terms used in this specification:
+
+- Project --- ...
```

Comment:
I don't think we can skip defining that, because it's such a central term.

"Project" is the only word I know for "the directory containing all the source files for a given program".

Reply:
Package may indeed be more accurate. Especially, if we could piggy-back on SPDX Spec here. @goneall, I can’t find where “Package” itself (as opposed to “Package Information”, “Package Name”, “Package Version”, etc.) is specified in SPDX Spec though. Am I missing something?

Reply:
I also looked for a definition in the spec and did not see a concise definition of a package.  I do recall discussing this at length during the spec development.  Something we may want to add to the spec.

Reply:
@goneall, in that case, I think it would be best if both projects use the same definition, until SPDX adopts it in a published spec. Then REUSE can simply refer to that definition.